src/clients/node/src/index.ts

```diff
@@ -1,3 +1,12 @@
+export * from './bindings'
```

Comment:
I need a TypeScript expert opinion here:
Is it the idiomatic way to import and re-export declarations?